mixins/localize-dynamic-mixin.js

```diff
@@ -7,6 +7,12 @@ export const LocalizeDynamicMixin = superclass => class extends LocalizeMixin(su
 
 	static async getLocalizeResources(langs, { importFunc, osloCollection }) {
 
+		// in dev, don't request unsupported langpacks
+		if (!importFunc.toString().includes('switch')) {
+			const supportedLangpacks = ['ar', 'cy', 'da', 'de', 'en', 'es-es', 'es', 'fi', 'fr-fr', 'fr', 'ja', 'ko', 'nl', 'pt', 'sv', 'tr', 'zh-tw', 'zh-cn'];
```

Comment:
I wonder if we could somehow leverage [these lists of supported locales](https://github.com/BrightspaceUI/intl/blob/main/lib/common.js#L3) from Intl... they're not exactly what you need though unfortunately. 🤔

Reply:
Similarly, we also have https://github.com/BrightspaceUI/actions/blob/master/prepare-translations-for-npm/prepare-translations.js#L4, https://github.com/BrightspaceUI/create/blob/main/src/generators/localization/templates/configured/_element.serge.json, https://github.com/Brightspace/serge-localize/blob/master/static/fra.common.serge.inc#L7... probably not for this PR to solve, but seems we could try to localize this to one source of truth.  For example, does each repo really need to define all this in their serge config, or can we just force a standard naming convection?  And could the `create` repo dynamically generate all the `lang` files based on the most up-to-date list? etc.